tools/kafka-versions-tools.sh

```diff
@@ -38,6 +38,65 @@ function get_kafka_versions {
 
 }
 
+function get_kafka_source_urls {
+
+    eval kafka_source_urls="$(yq read $VERSIONS_FILE '*.url' -j | tr '[],' '() ')"
+
+}
+
+function get_kafka_source_files {
+
+    eval kafka_source_files="$(yq read $VERSIONS_FILE '*.file' -j | tr '[],' '() ')"
+
+}
+
+function get_url_file_map {
+    # This function loops over the yaml objects for each version, rather the just using a single line 
+    # wild card, because the url and file entries are not guaranteed to always be present and 
```

Comment:
What happens when _the url and file entries are not present_?

Reply:
It errors out with 'exit 1'. See line 85.